engine/events/celebi.asm

```diff
@@ -159,9 +159,9 @@ UpdateCelebiPosition:
 	ld hl, SPRITEANIMSTRUCT_XCOORD
 	add hl, bc
 	add [hl]
-	cp 8 * 11 + 4
+	cp TILE_WIDTH * 11 + 4
 	jr nc, .ShiftY
-	cp 8 *  8 + 4
+	cp TILE_WIDTH *  8 + 4
```

Comment:
Good opportunity to *not* use that double-space alignment.

Reply:
Aaand done.